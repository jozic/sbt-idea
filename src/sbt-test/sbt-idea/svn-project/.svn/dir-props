K 10
svn:ignore
V 13
target
*.xml

END
