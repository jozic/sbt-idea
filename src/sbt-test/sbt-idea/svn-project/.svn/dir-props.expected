K 10
svn:ignore
V 36
target
*.xml

.idea/
.idea_modules/

END
